tests/sentry/issues/test_escalating.py

```diff
@@ -241,22 +234,22 @@ def test_daily_count_query(self) -> None:
         # The group had 3 events two days ago
         two_days_ago_mins = 48 * 60
         for i in range(4, 1, -1):
-            event = self._load_event_for_group(
-                fingerprint="group-query", minutes_ago=two_days_ago_mins + i
-            )
-            group = event.group
+            event = self._load_event_for_group(minutes_ago=two_days_ago_mins + i)
 
         # The group had 2 events yesterday
-        one_day_ago_mins = 24 * 60
+        # Tests that events are aggregated in the daily count query by date, not by 24 hr periods
+        yesterday = datetime.now().date() - timedelta(days=1)
+        yesterday_midnight = datetime.combine(yesterday, datetime.min.time())
+        mins_since_yesterday_midnight = int(
+            ((datetime.now() - yesterday_midnight).total_seconds()) / 60
+        )
         for i in range(3, 1, -1):
-            event = self._load_event_for_group(
-                fingerprint="group-query", minutes_ago=one_day_ago_mins + i
-            )
-            group = event.group
+            # Event occured i hours after yesterday midnight
+            event = self._load_event_for_group(minutes_ago=mins_since_yesterday_midnight + i * 60)
 
         # The group has 1 event today
         for i in range(2, 1, -1):
-            event = self._load_event_for_group(fingerprint="group-query", minutes_ago=i)
+            event = self._load_event_for_group(minutes_ago=i)
```

Comment:
This would still suffer from intermittency issues but let's move it to another PR.
I think `_load_event_for_group` should have a parameter being the reference datetime to create an event from.

Reply:
Yes, I agree. I think I misunderstood when you brought this up earlier. I'll open a new PR for this